File: code_blocks/integrations/third-party-integrations/airship_1.dart
import 'package:purchases_flutter/purchases_flutter.dart';
import 'package:airship_flutter/airship_flutter.dart';

Future<void> setAirshipChannel() async {
  await Purchases.configure(
    PurchasesConfiguration('public_sdk_key')..appUserId = 'my_app_user_id',
  );

  // Airship is configured in native code with your app key
  final channelId = await Airship.channelId;
  await Purchases.setAirshipChannelId(channelId);
}

File: code_blocks/integrations/third-party-integrations/airship_1.ts
import Purchases from '@revenuecat/purchases-react-native';
import Airship from 'urbanairship-react-native';

async function setAirshipChannel() {
  await Purchases.configure({
    apiKey: 'public_sdk_key',
    appUserID: 'my_app_user_id',
  });

  // Airship is configured in native code with your app key
  const channelId = await Airship.getChannelId();
  await Purchases.setAirshipChannelID(channelId);
}
